.rubocop.yml

```
@@ -1,14 +1,15 @@
 require: 'rubocop-rspec'
 
 AllCops:
-  TargetRubyVersion: 2.3
+  NewCops: enable
+  TargetRubyVersion: 2.6.9
```

```suggestion
  TargetRubyVersion: 2.6
```

I don't think the patch version will do anything?

---

🤷 it didn't complain 😅